### PR TITLE
Add `flake8-github` formatter

### PR DESCRIPTION
**Pull request by maxkrivich**
- Add a link to the flake8-github formatter

### CODE REVIEW

**Comment by maxkrivich** [comment]
Fixed the linter errors

File: README.md [hunk 1]
```diff
@@ -239,5 +239,6 @@ Extensions for formatting flake8 output.
 - [flake8-gl-codeclimate](https://github.com/awelzel/flake8-gl-codeclimate) - Generates Gitlab Code Quality artifacts.
 - [flake8-html](https://github.com/lordmauve/flake8-html) - Generates an HTML report.
 - [flake8-json](https://github.com/PyCQA/flake8-json) - Generates JSON output.
+- [flake8-github](https://pypi.org/project/flake8-github) - Generates GitHub annotation for PR's.
```
Comment (orsinium) [inline README.md:242]: Please, add a link to github page of the project rather than PyPI, as for all other packages in the list.

Comment (maxkrivich) [reply README.md]: Ohh, you're right. Thanks!

**Comment by maxkrivich** [comment]
> Thank you for your contribution! According to our [contribution guidelines](https://github.com/DmytroLitvinov/awesome-flake8-extensions/blob/master/contributing.md?rgh-link-date=2023-08-14T08%3A43%3A08Z), the package must be at least 30 days old before it can be added to the list. Let it boil for a while :)

Awesome. I will mark the date in my calendar to remind you. Thanks!

**Review by orsinium — APPROVED** [review]

**Comment by orsinium** [comment]
LGTM. When one more maintainer approves it, we can get it merged. Thank you for your contribution :)

**Comment by maxkrivich** [comment]
@DmytroLitvinov Could you please review and approve it when you have a moment? Your feedback is greatly appreciated.

**Comment by DmytroLitvinov** [comment]
LGTM 👍

**Review by DmytroLitvinov — APPROVED** [review]

**Pull request merged**